### PR TITLE
Update mining.md

### PR DESCRIPTION
**Pull request by jagottsicher**
minor changes to increase readability (easier copying) and a typo (echo needs double quotes if you want to add with >> append it to a file)

### CODE REVIEW

**Review by jcvernaleo — CHANGES_REQUESTED** [review]
I like fixing the copy and paste part but I don't think the double quotes on line two are needed unless I'm missing something.

File: docs/mining.md [hunk 1]
```diff
@@ -21,9 +21,9 @@ certificate into the default system Certificate Authority list.
 
 ## Ubuntu
 
-1. Copy rpc.cert to /usr/share/ca-certificates: `# cp /home/user/.btcd/rpc.cert /usr/share/ca-certificates/btcd.crt`
-2. Add btcd.crt to /etc/ca-certificates.conf: `# echo btcd.crt >> /etc/ca-certificates.conf`
-3. Update the CA certificate list: `# update-ca-certificates`
+1. Copy rpc.cert to /usr/share/ca-certificates: `cp /home/user/.btcd/rpc.cert /usr/share/ca-certificates/btcd.crt`
+2. Add btcd.crt to /etc/ca-certificates.conf: `echo "btcd.crt" >> /etc/ca-certificates.conf`
```
[the remark this answers is not in the document]
Comment (jagottsicher) [reply docs/mining.md:25]: TL;DR;
my bad. thank for merging/reviewing the rest. 

TL;DR;
clear case of "just a habit"!
but honestly: I grepped through my `.bash_history` to find out what made my thinking that the quotes were needed. I have a blurry idea I saw an error popping up. But it was a typo on my side (`btccd.cert` instead of `btcd.cer`t) hence my `update-ca-certificates` failed. Half-sleepy already must have misinterpreted that.

[the remark this answers is not in the document]
Comment (jcvernaleo) [reply docs/mining.md:25]: No worries.  I think so much of what we do with the shell comes down to habit and may or may not be necessary :)

Anyway, if you could remove the quotes I'll approve and merge.

**Comment by jagottsicher** [comment]
put the quotes in again, commmited, and this pull request caught it.

**Review by jcvernaleo — APPROVED** [review]
OK

**Comment by jcvernaleo** [comment]
Once the github actions finish running I'll merge.
